ui/mirage/factories/configuration.js

```diff
@@ -0,0 +1,27 @@
+import { Factory, trait } from 'ember-cli-mirage';
+import faker from 'faker';
```

Comment:
I don't see faker in package.json for 1.9 🤔 sort of an interesting scenario, because we typically don't touch dependencies in patch versions. Let's talk about this next week

Reply:
Ok sounds good. I just noticed that only the tests for 1.7.x are failing when trying to import faker but not on the other branches which is strange.

Reply:
Cherry picking the faker removal from #13778